common/djangoapps/util/tests/test_db.py

```diff
@@ -237,6 +237,6 @@ def test_migrations_are_in_sync(self):
         call_command('makemigrations', dry_run=True, verbosity=3, stdout=out)
         output = out.getvalue()
         # Temporary for `edx-enterprise` version bumps with migrations.
-        # Please delete when `edx-enterprise==0.48.3`.
+        # Please delete when `edx-enterprise==0.49.1`.
```

Comment:
@tuchfarber @douglashall Hopefully this PR won't cause issues in the midst of your safe deployment procedure?